website/tutorials.json

```diff
@@ -95,6 +95,14 @@
     "github": "https://github.com/facebookresearch/beanmachine/blob/v0.1.2/tutorials/Neals_funnel.ipynb",
     "colab": "https://colab.research.google.com/github/facebookresearch/beanmachine/blob/v0.1.2/tutorials/Neals_funnel.ipynb"
   },
+  "Bayesian_NNs_with_ADVI": {
```

Comment:
Can we put this under the ADVI tutorial, near the end with the other VI tutorials?